### PR TITLE
rust: update to 1.44

### PR DESCRIPTION
**Pull request by jnbr**
still testing ...

### CODE REVIEW

**Comment by q66** [comment]
what about cargo?

**Comment by jnbr** [comment]
can be done separately

**Comment by fosslinux** [comment]
Why `nocross` for now? Is that because the bootstrap tarballs need updating? :confused: 

Also typo in commit message "upddate".

File: srcpkgs/rust/template [hunk 1]
```diff
@@ -19,9 +19,10 @@ maintainer="Enno Boland <gottox@voidlinux.org>"
 license="MIT, Apache-2.0"
 homepage="https://www.rust-lang.org/"
 distfiles="https://static.rust-lang.org/dist/rustc-${version}-src.tar.gz"
-checksum=cde177b4a8c687da96f20de27630a1eb55c9d146a15e4c900d5c31cd3c3ac41d
+checksum="bf2df62317e533e84167c5bc7d4351a99fdab1f9cd6e6ba09f51996ad8561100"
```
Comment (fosslinux) [inline srcpkgs/rust/template:22]: Why quotes?

Comment (jnbr) [reply srcpkgs/rust/template]: no good reason, removed

**Comment by jnbr** [comment]
> Why nocross for now? Is that because the bootstrap tarballs need updating? confused

No, it's unrelated to bootstrap binaries. We skip one step when cross building rust but this requires to have the same version of rustc on the host (config option local-rebuild). So we disable cross builds, wait for native builds to complete and re-enable it again. It's a bit annoying but should save a lot of build time.

> Also typo in commit message "upddate".

thanks, fixed.

**Comment by fosslinux** [comment]
Ah makes sense.

**Pull request merged**